test/unit/gateways/visanet_peru_test.rb

```diff
@@ -90,14 +90,14 @@ def test_successful_refund
     response = @gateway.purchase(@amount, @credit_card, @options)
     assert_success response
 
-    @gateway.expects(:ssl_request).returns(successful_refund_response)
+    @gateway.expects(:ssl_request).with(:put, any_parameters).returns(successful_refund_response)
```

Comment:
You'd think that this would be `post` if you glanced quickly at the `method`…erm, method, but I actually can't find the genuine `refund` route tested, period; just `cancelDespoit`--and that's indeed a `put`.

Reply:
@bpollack Do you think it is beyond the scope of this PR to add tests for `prepare_refund_data`?

Reply:
_Probably_. But more practically, I just can't get to it today; I need to wrap up some other gateway work. So I'd either need someone else to add that test or we'll need to hold off merging this today. Since builds are technically "broken", I'd rather just go ahead and merge this.

Reply:
👍